README.md

```diff
@@ -5,25 +5,83 @@
 
 Update your dependencies, with hooks, for developers.
 
-## Run
+## Setup your repository
+
+First create a `lure.config` in the repository you want to keep your dependencies up-to-date.
+
+The file should look like:
+
+```{
+    "projects": [
+        {
+            "vcs": "hg",
+            "owner": "dreisch",
+            "name": "catfeederhg",
+            "defaultBranch": "default",
+            "branchPrefix": "lure-",
+            "commands": [
+                {
+                    "name": "updateDependencies"
+                }
+            ]
+        },
+        {
+            "vcs": "git",
+            "owner": "dreisch",
+            "name": "catfeedergit",
+            "defaultBranch": "master",
+            "branchPrefix": "lure-",
+            "commands": [
+                {
+                    "name": "synchronizedBranches",
+                    "args": {
+                        "from": "staging",
+                        "to": "develop"
+                    }
+                }
+            ]
+        }
+    ]
+}
+```
+
+Possible vcs are:
+- `hg` for mercurial
+- `git` for git
+
+The possible commands are:
+- `updateDependencies`
+- `synchronizedBranches`
+
+Other:
+- `owner`: https ://bitbucket.org/**owner**/name
+- `name`: https ://bitbucket.org/owner/**name**
+
+
+## Setup your CI
 
 eg, in jenkins:
 
-```sh
-wget https://github.com/coveo/Lure/releases/download/v1.0/lure-linux-amd64
-chmod +x lure-linux-amd64
-./lure-linux-amd64 -auth env -config <yourconfig>
+```env
+git config --global user.email "Youmail@example.com"
+git config --global user.name "jenkins"
+
+wget https://github.com/coveo/lure/releases/download/1.1.2/lure-linux-amd64 -O lure
+chmod +x lure
+./lure -auth env -config ${WORKSPACE}/lure.config
+
 ```
 
 You need bitbucket api-key and api-secret, see, the [bitbucket documentation](https://confluence.atlassian.com/bitbucket/oauth-on-bitbucket-cloud-238027431.html#OAuthonBitbucketCloud-OAuth2.0) for OAuth setup.
 
+Environment variables:
 - `IGNORE_DECLINED_PR=1` Will ignore declined PR when looking if the PR exists
 - `BITBUCKET_CLIENT_ID` the bitbucket OAuth **Key** previously created
 - `BITBUCKET_CLIENT_SECRET` the bitbucket OAuth **Secret** previously created
-- `BITBUCKET_REPO_NAME` https ://bitbucket.org/owner/**name**
-- `BITBUCKET_REPO_OWNER` https ://bitbucket.org/**owner**/name
 - `DRY_RUN` won't create a PR
 
+(Coveo employees take a look to https://jenkins.cloud.coveo.com/job/Reveal_Lure )
```

Comment:
👀 
well maybe who cares about the url but "reveal" is pretty forbidden for sure

Reply:
What do you mean... the jenkin's job purpose is to _reveal_ how to use lure. :trollface: